web/src/config/client_swagger.json

```diff
@@ -164,37 +125,25 @@
 				"summary": "Get CSM user list: CSM user having permission manage and monitor can get CSM users list",
 				"responses": {
 					"200": {
-						"description": "OK"
-					},
-					"201": {
-						"description": "No Content"
+						"description": "OK: List of CSM users"
 					},
 					"400": {
-						"description": "Bad Request"
+						"description": "Bad Request : Request body missing or S3 account with same name as passed CSM username already exists"
```

Comment:
This message "S3 account with same name as passed CSM username already exists" will appear only during creation of csm/user

Reply:
removed incorrect message and updated message